configure.py

```diff
@@ -928,6 +933,13 @@ def configure_node(o):
   o['variables']['want_separate_host_toolset'] = int(
       cross_compiling and want_snapshots)
 
+  if options.with_node_snapshot:
+    o['variables']['node_use_node_snapshot'] = 'true'
```

Comment:
we have the `b()` function for this
https://github.com/nodejs/node/blob/4c3c877fb206c02cf9fb8cd27612e9f09b8e9f9a/configure.py#L611-L616
but personally I'm Ok with this as is since it allows the explicit comment

Reply:
hmm, maybe this makes more sense?

```python
  if options.with_node_snapshot is None:
    o['variables']['node_use_node_snapshot'] = 'false'
    # When hashseed is fixed
    # o['variables']['node_use_node_snapshot'] = b(not cross_compiling)
  else:
    o['variables']['node_use_node_snapshot'] = b(options.with_node_snapshot)
```
`b(cross_compiling)` in the first branch?

Reply:
OK, we can leave it here until we fix the hash seed thing and come back turning this on.